contracts/interfaces/IInbox.sol

```diff
@@ -12,112 +12,26 @@ import {Route} from "../types/Intent.sol";
  * Hyperlane instant/batched)
  */
 interface IInbox is ISemver {
-    struct ClaimantAndBatcherReward {
-        address claimant;
-        uint96 reward;
-    }
-
     /**
      * @notice Emitted when an intent is successfully fulfilled
      * @param _hash Hash of the fulfilled intent
      * @param _sourceChainID ID of the source chain
+     * @param _prover Address of the prover that fulfilled the intent
      * @param _claimant Address eligible to claim rewards
```

Comment:
The events and error pattern has changed significantly. Consider maintaining some backward compatibility for event listeners to avoid breaking existing monitoring systems.